hapi/hedera-protobufs/services/state/blockstream/block_stream_info.proto

```diff
@@ -37,7 +37,12 @@ option java_package = "com.hederahashgraph.api.proto.java";
 option java_multiple_files = true;
 
 /**
- * A message stored in state to maintain block stream parameters.
+ * A message stored in state to maintain block stream parameters.<br/>
+ * Nodes use this information for three purposes.
+ * 1. To maintain hash chain continuity at restart and reconnect boundaries.
+ * 1. To store historical hashes for implementation of the EVM `BLOCKHASH`
+ *    and `PREVRANDAO` opcodes.
+ * 1. To track the amount of consensus time that has passed between blocks.
```

Comment:
Minor: you may want to renumber the items in the list, or use an unnumbered list instead. Using the same number looks confusing for anyone reading this source file directly.

Reply:
I think the repeated `1.` is the convention @jsync-swirlds is using in the protobufs.

Reply:
I see. That's a bit strange as it makes it confusing when editing those files in a text editor. But okay.